src/index.html

```diff
@@ -46,7 +46,14 @@ <h1>Instant, themed Twitch chat overlays</h1>
 			<a href="docs/configuration/#show-latest-messages" aria-label="Docs for showLatestMessages">&#9432;</a>
 		</div>
 		<input type="number" id="show-latest-messages" name="showLatestMessages"
-			value="{{defaults.showLatestMessages}}" />
+			placeholder="{{defaults.showLatestMessages}}" />
+
+		<div class="overlay-builder-header">
+			<label for="clear-message-after">Clear messages after (seconds)</label>
+			<a href="docs/configuration/#clear-message-after" aria-label="Docs for clearMessageAfter">&#9432;</a>
+		</div>
+		<input type="number" id="clear-message-after" name="clearMessageAfter"
+			placeholder="{{defaults.clearMessageAfter}}" step="0.1" />
```

Comment:
```suggestion
			placeholder="{{defaults.clearMessageAfter}}" step="0.1" min="0"/>
```
Preventing negative numbers from being entered clientside.